Gemfile

```diff
@@ -13,6 +13,7 @@ gem "jekyll", "~> 4.0.0"
 # If you have any plugins, put them here!
 group :jekyll_plugins do
   gem "jekyll-feed", "~> 0.12"
+  gem "octopress-debugger"
```

Comment:
this is useful when you want to find out what variables are available in a given context, check how to use it here: https://github.com/octopress/debugger